### PR TITLE
MINIFICPP-1546 Added documentation for enabling OPC functionality at build time

### PR DESCRIPTION
**Pull request by jdye64**
OPC was missing from the cmake flag table so I added it.

### CODE REVIEW

**Comment by adamdebreceni** [comment]
there are quite a number of changes that seem to be unrelated to the PR, removed documentations, removed options, added dependencies, some typos, changed version numbers

**Comment by jdye64** [comment]
Yeah don't know what happened there? Let me make another commit

**Comment by szaszm** [comment]
It looks like you made your changes to an old version of README.md

**Review by adamdebreceni — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -617,4 +618,4 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License.
+limitations under the License.
```
Comment (szaszm) [inline README.md:621]: Please keep the line break at the end of the file

Comment (adamdebreceni) [reply README.md:621]: is that the change? I used to see a little icon if the \n was the diff 🚫

Comment (szaszm) [reply README.md]: I checked out the branch locally to see the difference and it looks like that. I remember the icons, too, but it looks like they are gone.

**Review by szaszm — APPROVED** [review]

**Review by lordgamez — APPROVED** [review]

**Comment by fgerlits** [comment]
Thanks for your contribution!  Would you mind creating a MINIFICPP-xxxx Jira for it, please?  We require a Jira for every pull request.

**Comment by jdye64** [comment]
Has it not always been minifi policy to no require JIRA's for single line documentation updates? Either way, yeah I'll make one

**Comment by jdye64** [comment]
This closes #https://issues.apache.org/jira/browse/MINIFICPP-1546

**Comment by joewitt** [comment]
@jdye64 as a general rule you're probably right but as the community grows those short cuts get harder to keep :)

**Comment by jdye64** [comment]
> @jdye64 as a general rule you're probably right but as the community grows those short cuts get harder to keep :)

That is great news! Love to see the community grow. Opened the JIRA as suggested.

**Pull request closed without merging**